docs/_assets/attributes.adoc

```diff
@@ -21,25 +21,32 @@
 :toclevels: 2
 :icons: font
 
-//Latest version
-:KroxyliciousVersion: 0.13
-:gitRef: tree/main
+// Latest version
+:KroxyliciousVersion: 0.14.0-SNAPSHOT
+// Used for links to release artifacts, will be 404 during development
+:KroxyliciousReleaseTag: v{KroxyliciousVersion}
+// Used to point at source code and blobs, will point at main during development and at the release tag when deployed
+:KroxyliciousGitRef: main
 :ApicurioVersion: 2.6.x
 :KubernetesVersionMinimum: 1.31
 :OpenShiftVersionMinimum: 4.18
 
-:OperatorDownloadUrl: https://github.com/kroxylicious/kroxylicious/releases[GitHub releases page^]
 :OperatorAssetZipFileName: kroxylicious-operator-{KroxyliciousVersion}.zip
 :OperatorAssetTgzFileName: kroxylicious-operator-{KroxyliciousVersion}.tar.gz
 
 //Proxy links
 :github: https://github.com/kroxylicious/kroxylicious
-:github-releases: https://github.com/kroxylicious/kroxylicious/{gitRef}
-:github-issues: https://github.com/kroxylicious/kroxylicious/issues
-:api-javadoc: https://javadoc.io/doc/io.kroxylicious/kroxylicious-api/{KroxyliciousVersion}
-:kms-api-javadoc: https://javadoc.io/doc/io.kroxylicious/kroxylicious-kms/{KroxyliciousVersion}
-:encryption-api-javadoc: https://javadoc.io/doc/io.kroxylicious/kroxylicious-encryption/{KroxyliciousVersion}
-:start-script: https://github.com/kroxylicious/kroxylicious/blob/{gitRef}/kroxylicious-app/src/assembly/kroxylicious-start.sh
+:github-release: {github}/releases/{KroxyliciousReleaseTag}
+:github-source-tree-root: {github}/tree/{KroxyliciousGitRef}
+:github-blob-root: {github}/blob/{KroxyliciousGitRef}
+:github-raw-root: https://raw.githubusercontent.com/kroxylicious/kroxylicious/{KroxyliciousGitRef}
+:OperatorDownloadUrl: {github-release}[GitHub releases page^]
```

Comment:
nit: Putting my pedant hat on, is operator-download really a `proxy link`?

Reply:
didn't notice that comment, a lot of these links aren't proxy specific will shuffle them around